cmake/pfs-config.cmake.in

```diff
@@ -0,0 +1,3 @@
+@PACKAGE_INIT@
+include("${CMAKE_CURRENT_LIST_DIR}/pfsTargets.cmake")
```

Comment:
It seems to me that this line should be removed.
When I tried to run this locally, it didn't work with this line because `pfsTargets.cmake` is not generated while `make`-ing or `make install`-ing.
When I removed this line, everything worked just fine.